### PR TITLE
Fixes #17055: Add systemd hardening config for slapd

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/17055

### CODE REVIEW

File: rudder-webapp/SOURCES/Makefile [hunk 1]
```diff
@@ -191,6 +191,7 @@ install: build initial-promises initial-ncf
 	install -m 644 rudder-sources/rudder/rudder-core/src/main/resources/ldap/rudder.schema $(DESTDIR)/opt/rudder/etc/openldap/schema/
 	install -m 644 rsyslog/rudder-slapd.conf $(DESTDIR)/etc/rsyslog.d/rudder-slapd.conf
 	install -m 644 rudder-slapd.service $(DESTDIR)/usr/lib/systemd/system/rudder-slapd.service
+	[ $$(systemctl --version | head -1 | cut -d' ' -f2) -gt 232 ] && sed -i '/\[Service\]/ r rudder-slapd-hardening' $(DESTDIR)/usr/lib/systemd/system/rudder-slapd.service || true
```
Comment (peckpeck) [inline rudder-webapp/SOURCES/Makefile:194]: where is rudder-slapd-hardening ?

Comment (amousset) [reply rudder-webapp/SOURCES/Makefile:194]: in my local repo

Comment (peckpeck) [reply rudder-webapp/SOURCES/Makefile:194]: does the systemctl command work within a chroot ?

**Comment by amousset** [comment]
PR updated with a new commit

File: rudder-webapp/SOURCES/Makefile (hunk 1 above)
Comment (peckpeck) [reply rudder-webapp/SOURCES/Makefile:194]: hehe

Comment (amousset) [reply rudder-webapp/SOURCES/Makefile:194]: we already use this for the relayd package

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**